src/util/check.js

```diff
@@ -0,0 +1,54 @@
+const pify = require('pify')
```

Comment:
Sorry, it's been a little while, but why does this file exist? It doesn't appear to be in use anywhere in the project (other than tests)...

Reply:
Sorry but this is a node package isn't it?

Reply:
Sorry, I should have been more clear. I'm commenting on the file `src/utils/check.js` not `pify`.

Reply:
sorry for being late. 
It seems that this file was removed in this https://github.com/jfmengels/all-contributors-cli/commit/152a1fe30a443ec5689ee0626e70a33571f06ce2 commit. I retested everything